todo_app/templates/index.html

```diff
@@ -7,12 +7,24 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <form action="add_todo_item" method="post">
+    </div>
```

Comment:
These `</div>` elements are the reason your alignment is working out oddly - remember that `<element>` opens a block, and `</element>` closes it. So this is accidentally closing the most recently opened div, which happens to be a part of the template that is handling putting some space round the outside, and the bits after aren't quite in the right place

Depending on exactly what you want, I'd probably suggest:
* Line 11 - delete this `</div>`
* Line 14 (now line 13 if you've already done the above) - remove the `/` so that opens a new container for the button

Possibly clear but in terms of Git, if you want to make these changes the easiest thing to do is:
* Make the change on your exercise-1 branch as you have been
* Add/commit/push that to GitHub once you're happy
* This PR will automatically update
* Merge it when you're ready (no need to ask for another review)
Or some people prefer to just consider any suggested changes alongside the next exercise, if you prefer that approach

Reply:
Thanks for the clear instruction Jack. Appreciate it!